bigquery_storage/synth.py

```diff
@@ -130,4 +130,10 @@
 )
 # END: Ignore lint and coverage
 
+# ----------------------------------------------------------------------------
+# Add templated files
+# ----------------------------------------------------------------------------
+templated_files = common.py_library(unit_cov_level=79, cov_level=79, samples_test=True, system_test_dependencies=["fastavro", "pandas", "pyarrow != 0.14.0"])
+s.move(templated_files)
```

Comment:
Might need one or more excludes here.

Reply:
I'd rather keep the common templates if possible. It seems to be working.